src-ui/src/app/components/app-frame/app-frame.component.ts

```diff
@@ -78,17 +76,21 @@ export class AppFrameComponent implements OnInit, OnDestroy {
   }
 
   closeAll() {
-    this.closeMenu()
-    this.openDocumentsService.closeAll()
-
-    // TODO: is there a better way to do this?
-    let route = this.activatedRoute
-    while (route.firstChild) {
-      route = route.firstChild
-    }
-    if (route.component == DocumentDetailComponent) {
-      this.router.navigate([""])
-    }
+    // user may need to confirm losing unsaved changes
+    this.openDocumentsService.closeAll().subscribe(confirmed => {
```

Comment:
Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/jonaswinkler/paperless-ng/pullRequest?prid=7592507)
```suggestion
    this.openDocumentsService.closeAll().subscribe((confirmed) => {
```